.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[src/**.java]
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
```

Comment:
Fixed. Actually, I was testing if auto-indent was working after changing editorconfig.
Forgot to rollback. 😃